build-dist.xml

```diff
@@ -2552,6 +2552,26 @@ this task.
 						value="${env.JAVA_HOME}"
 					/>
 				</replace>
+
+				<chmod
```

Comment:
This can be done with one chmod task containing two filesets.

    <chmod perm="a+x" >
        <fileset dir="${app.server.weblogic.bin.dir}" includes="**/*.sh" />
        <fileset dir="${app.server.weblogic.instance.dir}" includes="**/*.sh" />
    </chmod>